src/main/java/org/testng/reporters/JUnitReportReporter.java

```diff
@@ -185,6 +178,50 @@ public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites,
 
   }
 
+  private TestTag createIgnoredTestTagFor(ITestNGMethod method) {
```

Comment:
Nothing important, but `static` (like `final` IMO) helps the reading: with a very quick look, you know that no external variables are used (and you can move this method without any problem if you want).
Moreover, too many static methods may highlight a bad design.
But again, it is just my personal point of view/taste ;)